### ISSUE TITLE
Error after upgrading from v1.5.4.13 to v1.5.4.17: Error: Process failed to start: The system cannot find the file specified.

### ISSUE THREAD

**Issue by R-P-H** [QuasarApp/CQtDeployer#704]
After upgrading to the latest version I get the error below using the following options:

`"cqtdeployer.bat" -bin "bin","bin\config.ini" -qmake C:\Qt\5.15.2\mingw81_64\bin\qmake.exe -targetDir "\Deployed" noTranslations -name "My Installer" -description "My Installer" -deployVersion 1.0 -releaseDate 2022-07-22 -publisher ME -icon "images\icon.ico" qif -qifLogo "images\my_logo.png" -installDirQIFW C: -qifOut "My Installer" noHashSum clear`

```
Error: Process failed to start: The system cannot find the file specified.
Error: Process error code: 0
Error: The CQtDeployer fail to Pack application.
```

**Comment by EndrII** [comment]
Looks as cqtdeployer can not find the qt install framework qifw   please check qt installer framework in your work PC.

You can install it using the cqtdeployer installer tool

**Comment by R-P-H** [comment]
> Looks as cqtdeployer can not find the qt install framework qifw please check qt installer framework in your work PC.
> 
> You can install it using the cqtdeployer installer tool

Hi, it is installed. The only thing I did was to update both compoenents (qt installer framework and cqtdeployer v1.5) and then I got the issue. I did not change anything else.

**Comment by EndrII** [comment]
Hm, 
maybe you can show me all deploy log with -verbose 3 option ?

**Comment by 1zun4** [comment]
This is due to the QtDeployer being installed at "C:\Program Files (x86)", which will cause issues not able to find the `binarycreator.exe`. After installing it to for example "C:\Qt" it worked fine. The issue is caused by the white space in the file path.

**Comment by R-P-H** [comment]
> This is due to the QtDeployer being installed at "C:\Program Files (x86)", which will cause issues not able to find the `binarycreator.exe`. After installing it to for example "C:\Qt" it worked fine. The issue is caused by the white space in the file path.

Thanks, I didn't get a chance to debug it as requested. Will this be fixed in a new version ?

**Comment by EndrII** [comment]
> > This is due to the QtDeployer being installed at "C:\Program Files (x86)", which will cause issues not able to find the `binarycreator.exe`. After installing it to for example "C:\Qt" it worked fine. The issue is caused by the white space in the file path.
> 
> Thanks, I didn't get a chance to debug it as requested. Will this be fixed in a new version ?

@R-P-H 
If @1zun4 is right, then you can fix your issue if you add the binarycreator executable into PATH variable.

**Comment by R-P-H** [comment]
> > > This is due to the QtDeployer being installed at "C:\Program Files (x86)", which will cause issues not able to find the `binarycreator.exe`. After installing it to for example "C:\Qt" it worked fine. The issue is caused by the white space in the file path.
> > 
> > 
> > Thanks, I didn't get a chance to debug it as requested. Will this be fixed in a new version ?
> 
> @R-P-H If @1zun4 is right, then you can fix your issue if you add the binarycreator executable into PATH variable.

Thanks, I will give it a try.

**Comment by EndrII** [comment]
This issue fixed on the cqtdeployer 1.6 (dev branch)
